packages/orchestration/src/orchestration-completion-from-json.test.ts

```diff
@@ -0,0 +1,131 @@
+import { constructCompletionFromJson } from './orchestration-client.js';
+
+describe('constructCompletionFromJSON()', () => {
```

Comment:
```suggestion
describe('construct completion from JSON', () => {
```

Reply:
[pp] This is where I feel a bit lost when seeing `construct completion`. It actually construct completion post request body. Find a way to make it more clear.